doc/api-extensions.md

```diff
@@ -2216,3 +2216,12 @@ With this mode enabled the following changes are made to the network:
 ## `ovn_nic_acceleration_vdpa`
 
 This updates the `ovn_nic_acceleration` API extension. The `acceleration` configuration key for OVN NICs can now takes the value `vdpa` to support Virtual Data Path Acceleration (VDPA).
+
+## `cluster_healing`
```

Comment:
Should we mention this is only for remote shared storage pools?

Reply:
Yeah, would make sense to mention that.